crates/sui-core/src/authority_batch.rs

```diff
@@ -231,21 +234,37 @@ impl crate::authority::AuthorityState {
         &self,
         request: BatchInfoRequest,
     ) -> Result<impl Stream<Item = Result<BatchInfoResponseItem, SuiError>>, SuiError> {
+        let metrics = self.metrics.clone();
+        metrics.follower_connections.inc();
```

Comment:
BTW, `follower_connections` should really be a gauge.  It goes up and down.
Counters are designed for things which only increase, and has logic in Prom and functions that accompany it (like rate) which assume that it only ever increases.